pkg/platform/runtime/buildscript/buildscript_test.go

```diff
@@ -73,10 +88,12 @@ in:
 			},
 		},
 		&In{Name: ptr.To("runtime")},
+		expr,
 	}, script)
 }
 
-func TestComplex(t *testing.T) {
+// Re-enable when multiple solves are supported in DX-2238.
+func NoTestComplex(t *testing.T) {
```

Comment:
Sorry to be nitpicky but can we do this with a `t.Skip()` for now? It just makes it more explicit imo.

Reply:
Good call!